### PR TITLE
fix: verify id for v1 entries

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

File: src/legacyIpfs3boxAccessController.js [hunk 1]
```diff
@@ -34,11 +34,18 @@ class LegacyIPFS3BoxAccessController {
     const publicKey = entry.v === 0 ? entry.key : await publicKeyFromDID(entry.identity.id)
     if (this.write.includes(publicKey) ||
       this.write.includes('*')) {
-      return true
+      return entry.v === 0 ? true : await identityProvider.verifyIdentity(entry.identity)
```
Comment (oed) [inline src/legacyIpfs3boxAccessController.js:37]: This makes sense! Was thinking about this fix today :D

File: src/legacyIpfs3boxAccessController.js [hunk 2]
```diff
@@ -34,11 +34,18 @@ class LegacyIPFS3BoxAccessController {
...
     }
     return false
   }
 
+
+  const publicKey = entry.v === 0 ? entry.key : entry.identity.id
+ if (this.write.includes(publicKey) ||	    if (this.write.includes(publicKey) || this.write.includes('*')) {
+   this.write.includes('*')) {	      return entry.v === 1 ? await identityProvider.verifyIdentity(entry.identity) : true
+   return true
+ }	    }
```
Comment (oed) [inline src/legacyIpfs3boxAccessController.js]: This code shouldn't be here?

Comment (zachferland) [reply src/legacyIpfs3boxAccessController.js]: gosh no, was referencing code from my orbitd pr, just removed

Comment (oed) [reply src/legacyIpfs3boxAccessController.js]: xD

**Review by oed — APPROVED** [review]

**Pull request merged**